CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Unreleased
+# 47.3.1
```

Comment:
I think this should be version `47.4.0` actually since it's a new non-breaking change.